antelope_contracts/contracts/erc20/src/erc20.cpp

```diff
@@ -334,4 +334,43 @@ void erc20::removeegress(const std::vector<name>& accounts) {
             egresslist_table.erase(it);
 }
 
+void erc20::withdrawfee(eosio::name token_contract, eosio::asset quantity, eosio::name to, std::string memo) {
+    require_auth(get_self());
+
+    uint128_t v = token_contract.value;
+    v <<= 64;
+    v |= quantity.symbol.code().raw();
+
+    token_table_t token_table(_self, _self.value);
+    auto index = token_table.get_index<"by.symbol"_n>();
+    auto itr = index.find(v);
+
+    eosio::check(itr != index.end() && itr->fee_balance.symbol == quantity.symbol, "token not registered");
```

Comment:
Again, split the conditions with separate error messages as discussed for `setingressfee`.